web/src/lib/components/shared-components/upload-panel.svelte

```diff
@@ -41,7 +41,7 @@
         });
       } else if ($successCounter > 0) {
         notificationController.show({
-          message: 'Upload success, refresh the page to see new upload assets.',
+          message: `Uploaded ${$successCounter} assets successfully, refresh the page to see new upload assets.`,
```

Comment:
I think it would be nice if we could conditionally chose `assets` or `asset` (no @jrasm91, we don't bring `s()` to web.)

Reply:
Ah good point, in keeping with the error statement above - will update now.